CMakeLists.txt

```diff
@@ -462,6 +462,8 @@ if(NOT STUMPLESS_WINDOWS_EVENT_LOG_TARGETS_SUPPORTED)
     SOURCES ${PROJECT_SOURCE_DIR}/test/function/config/wel_unsupported.cpp
   )
 else()
+  set(STUMPLESS_WINDOWS_EVENT_LOG_TARGETS_SUPPORTED TRUE)
+
```

Comment:
you can remove these lines as well - they were changed in a recent commit (only a few days ago), so I suspect they are just left in your branch from the older version